src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -847,6 +848,7 @@ bool PerformInteractionRequest::IsWhiteSpaceExist() {
   LOG4CXX_AUTO_TRACE(logger_);
   const char* str = NULL;
 
+  // cppcheck-suppress redundantAssignment
   str = (*message_)[strings::msg_params][strings::initial_text].asCharArray();
```

Comment:
const char* str = (*message_)[strings::msg_params][strings::initial_text].asCharArray();

And in other places

Reply:
Fixed in d4c6de3
Please check